pgbackrest_auto

```diff
@@ -647,16 +661,22 @@ if [[ "${CHECKDB_MODE}" != "No" ]]; then
     if [[ "${CHECKDB}" = "yes" || "${CHECKSUMS}" = "yes" ]]; then
     info "[STEP $((STEP++))]: Verify data checksums"
     pg_checksums
+    else
+    sed -i 's/PG_checksums_validation=0/PG_checksums_validation=1/g' ${zbx_status_file}
     fi
     # verify that data can be read out. Check with pg_dump >> /dev/null
     if [[ "${CHECKDB}" = "yes" || "${DUMMYDUMP}" = "yes" ]]; then
     info "[STEP $((STEP++))]: Verify that data can be read out"
     dummy_dump
+    else
+    sed -i 's/Data_validation=0/Data_validation=1/g' ${zbx_status_file}
     fi
     # amcheck - verify the logical consistency of the structure of PostgreSQL B-Tree indexes
     if [[ "${CHECKDB}" = "yes" || "${AMCHECK}" = "yes" ]]; then
     info "[STEP $((STEP++))]: Verify indexes"
     amcheck
+    else
+    sed -i 's/Amcheck_validation=0/Amcheck_validation=1/g' ${zbx_status_file}
```

Comment:
this seems redundant, since the amcheck function also writes statuses

Reply:
before new restore all values resets in zabbix status file (line 602)